workers/constructors/SharedWorker/interface-objects.html

```diff
@@ -14,7 +14,7 @@
   worker.port.start();
   worker.port.postMessage(supported);
   worker.port.onmessage = t.step_func_done(e => {
-    assert_equals(e.data, '');
+    assert_equals(e.data, 'These were missing: SharedWorker');
```

Comment:
This change should probably be a in a separate PR? Also rather than changing the assert here, I think removing SharedWorker from the expected list on line 7 would make more sense.

Reply:
(while you're at it you might also want to remove ApplicationCache from that list, as that API isn't in the spec anymore)

Reply:
@mkruisselbrink Thanks for the great feedback! I put the change in a [separate pull request](https://github.com/web-platform-tests/wpt/pull/29988/files) as you advised and applied your suggestions.